webapp/app/css/treeherder.css

```diff
@@ -1808,6 +1808,20 @@ fieldset[disabled] .btn-repo.active {
     background-color: #fff;
 }
 
+#help-job-name {
+    color: #428bca;
+}
+
+#help-raw-log-icon {
+    font-size: 13px;
+    color: #9fa3a5;
+}
+
+#help-logviewer-icon {
+    vertical-align: text-bottom;
+    width: 15px;
+}
+
```

Comment:
Re-using the icons in Help with their existing treeherder classes was a bit problematic, so these ids were added.